widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadLayout.java

```diff
@@ -102,12 +102,12 @@ public boolean isInChatView() {
 
     @Override
     public void show() {
-        this.setVisibility(VISIBLE);
+        post(() -> setVisibility(VISIBLE));
```

Comment:
there seems to be a deeper problem with threading. current solution to patch the crashes by adding `post` to the different places inside the views is not very scalable.